README.md

```diff
@@ -302,6 +203,117 @@ Applies only to `install-wcp.bash`
 | MYSQL_PASSWORD            | MySQL user's password (must meet complexity standards)                                                                               | NULL                                                                                                         | yes      |
 | MYSQL_ROOT_PASSWORD       | MySQL "root" user's password (must meet complexity standards)                                                                        | NULL                                                                                                         | yes      |
 
+
+
+## Invocation
+
+To avoid the complexity of parsing CLI flags in bash or with `getopts`, and to maximize the portability of the script, all inputs to the script should be supplied as Environment Variables. So invocation is expected to look like:
+
+```bash
+LABKEY_COMPANY_NAME='Megadodo Publications' ... ./install-labkey.bash
+```
+
+And hopefully in the not-to-distant future:
```

Comment:
hopefully when? If it doesn't exist yet and isn't in a planned release, should we take this section out?

otherwise there's a missing o:  not-to-distant > not-too-distant

Reply:
Thanks I thought I removed this...